### PR TITLE
fix: testing and linting on windows

### PR DESCRIPTION
**Pull request by mdonnalley**
Make fixes to be able to run unit tests in windows

@W-8623693@

### CODE REVIEW

**Review by WillieRuemmele — APPROVED** [review]

File: packages/prettier-config/prettier-config.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
   singleQuote: true,
   printWidth: 120,
+  endOfLine: 'auto',
```
Comment (amphro) [inline packages/prettier-config/prettier-config.js:4]: Why is this needed to run tests on windows?

**Review by shetzel — APPROVED** [review]

File: packages/prettier-config/prettier-config.js (hunk 1 above)
Comment (mdonnalley) [reply packages/prettier-config/prettier-config.js:4]: @amphro without it linting fails because windows uses a different EOL character than linux and mac: https://eslint.org/docs/rules/linebreak-style
